nservicebus/azure/intro-keep-connection-string-safe.include.md

```diff
@@ -0,0 +1,4 @@
+It is possible to accidentally leak sensitive information in the connection string if it's not properly secured. E.g. the information can be leaked if an error occurs when communicating across untrusted boundaries, or if the error information is logged to an unsecured log file.
```

Comment:
After reading this I'm still not sure how this affects me as an NSB user. Does it mean the log files are not sure? What are the boundaries that the reference made to?